spec/factories/projects.rb

```diff
@@ -1,8 +1,8 @@
 FactoryBot.define do
   factory :project do
-    name { "Conveyor Belt" }
-    user { staff }
-    hash_id { "abc123" }
-    project_board_base_url { "https://github.com/turingschool-examples/watch-and-learn/projects/1" }
+    name { Faker::Company.catch_phrase }
+    user { create(:staff) }
```

Comment:
does FactoryBot now require us to call `create` here instead of just the name of the factory? never seen this before.

Reply:
I think you can use the name of the factory outside of the block, but not within the block. The error I get is `undefined method staff`.